### PR TITLE
Added Governance

### PR DESCRIPTION
**Pull request by mcollina**

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+GitHub regarding that and reply back when a decision has been made.
+
+The consensus to grant a new candidate Collaborator status is reached when:
+
+- at least one of the Lead Maintainers approve
+- at least two of the Team Members approve
+
+## Lead Maintainers nominations
+
+A Team Member may be promoted to a Lead Maintainers only through nomination by a
```
Comment (wesleytodd) [inline GOVERNANCE.md:74]:
```suggestion
A Team Member may be promoted to a Lead Maintainer only through nomination by a
```

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+Lead maintainer and with agreement from the rest of Lead Maintainers.
```
Comment (wesleytodd) [inline GOVERNANCE.md:75]:
```suggestion
Lead Maintainer and with agreement from the rest of Lead Maintainers.
```

File: GOVERNANCE.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+Great contributors on a specific area in the Neostandard ecosystem will be invited
+to join this group by Lead Maintainers. This group has the same permissions of a
+contributor.
+
+## Collaborator nominations
+
+Individuals making significant and valuable contributions to the project may be
+a candidate to join the Neostandard organization.
+
+A Collaborator needs to open a private team discussion in [`neostandard/collaborators`](https://github.com/neostandard/collaborators) and list the
```
Comment (wesleytodd) [inline GOVERNANCE.md:55]: This repo does not yet exist, should we create it now?

File: GOVERNANCE.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+- at least two of the Team Members approve
+
+## Lead Maintainers nominations
+
+A Team Member may be promoted to a Lead Maintainers only through nomination by a
+Lead maintainer and with agreement from the rest of Lead Maintainers.
+
+## Consensus seeking process
+
+The Neostandard organization follows a Lazy [Consensus Seeking][] decision-making model,
```
Comment (wesleytodd) [inline GOVERNANCE.md:79]: If we want to reference the OpenJS CPC version of this? https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-9-decision-making

**Review by wesleytodd — APPROVED** [review]
Just small nit picks, otherwise LGTM

File: GOVERNANCE.md (hunk 3 above)
Comment (voxpelli) [reply GOVERNANCE.md:55]: Created it now 🙏

**Comment by voxpelli** [comment]
@wesleytodd @LinusU @bcomnes @mcollina: Are we ready to merge?

**Review by bcomnes — APPROVED** [review]
Lgtm.

**Comment by rhettjay** [comment]
I'm curious the appeal of moving this effort to an {org}/.github repo to include the ability to have issue templates etc, but maybe it's a tad preemptive. Is there hope/intent the project will evolve beyond the base linter perhaps to include perhaps code editor extensions (like vscode, neovim, etc.)? If so, a .github may be worth consideration.

**Comment by bcomnes** [comment]
Open to .github when it makes sense or someone wants to do it. Issue templates are an impediment/speed bump to contributing and I want to keep that a barrier as low as possible for now. 

One of the goals here is to reduce the implementation burden around integration plugins and rely on eslint LSP directly which already has good ecosystem support for almost every editor. This also solves the problem of contributors walking away from plugin maintenance. There are other tangential standard projects like snippet collections we could adopt though.

**Comment by mcollina** [comment]
@voxpelli this can land!

**Pull request merged**